roadmap/implementers-guide/src/node/disputes/dispute-coordinator.md

```diff
@@ -9,7 +9,7 @@ In particular the dispute-coordinator is responsible for:
 
 - Ensuring that the node is able to raise a dispute in case an invalid candidate
   is found during approval checking.
-- Ensuring malicious approval votes will be recorded, so nodes can get slashed
+- Ensuring lazy approval votes will be recorded, so nodes can get slashed
```

Comment:
nit: Might be worthwhile to clarify what "lazy" means. I got the idea by reading the section below, but it would be nice to define it.

Reply:
Fixed this. Good call out!